pkg/signalmeow/groups.go

```diff
@@ -494,3 +593,130 @@ type GroupCache struct {
 	lastFetched map[types.GroupIdentifier]time.Time
 	activeCalls map[types.GroupIdentifier]string
 }
+
+func (cli *Client) GetGroupChange(ctx context.Context, groupContext *signalpb.GroupContextV2, gid types.GroupIdentifier) (*GroupChange, error) {
```

Comment:
This should probably be called `DecryptGroupChange` since it isn't getting anything other than the authorization (why is it getting the authorization anyway? 🤔)

Reply:
I wasn't sure whether getting authorization is necessary. It's changed now.